brianlong/www.validators.app
# 187115687 enchance stake account gathering

#### What's this PR do?
- stake accounts are gathered once per epoch
- schedule gathering script every hour
- rescue from timeouts in explorer stake accounts

#### How should this be manually tested?
- run gather_stake_accounts.rb script
- it should process accounts only once per epoch
- run tests

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/187115687)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
